src/cljs/rems/fields.cljs

```diff
@@ -59,6 +73,8 @@
   :field/optional - boolean, true if the field is not required
   :field/value - string, the current value of the field
   :field/previous-value - string, the previously submitted value of the field
+  :field/info-text - text for collapsable info field
+  :field/collapse-aria-label - text for aria-label of info button
```

Comment:
I addressed that in my code 
But I agree, maybe I can refactor

Reply:
I believe this one is resolved

Reply:
yeah

Reply:
I originally meant that you could possibly remove the parameter altogether and just have "More info" as the text. Not 100% sure on the accessibility of it but it would remove the need to pass or maintain them.

Reply:
well, this is outdated, the only thing it has to do is to put in some code and we are already putting in a string here

Reply:
Yes I'm just wondering that could this have been left out completely and have always had the localized "More info". It's anyway marginally better to have "More info about foobar" in the text although `text-format` should be used and not string concatenation. We can leave it as it is.